Source/m_random.h

```diff
@@ -118,6 +118,9 @@ typedef enum {
   pr_dropoff,                 // #60
   pr_randomjump,              // #61
   pr_defect,                  // #62
+  // mbf21
+  pr_heretic,                 // #63
```

Comment:
I don't think the index in this list matters, so you could just as well skip `pr_heretic` if you want. MBF21 doesn't support splitting the rng between all these entries - in practice there is only one pool because demo_insurance is always 0. I added `pr_heretic` to dsda-doom for example purely as a self-documenting marker since it didn't matter one way or the other.

Reply:
Good to know, thanks.